### DISCUSSION TITLE
Node.js 16 End of Life

### DISCUSSION THREAD

**Discussion by amcintosh** [freshbooks/freshbooks-nodejs-sdk#1010] [General]
PSA that Node.js 16 was EOL as of [September 11, 2023](https://nodejs.org/en/about/releases/).

We will be dropping support for it soon in a new major release due to some of our dependencies dropping support.

**Comment by calvin-fb** [comment]
FYI @ngalandefb

**Comment by iietmoon** [comment]
@ngalandefb @calvin-fb Node.js has an incredible ecosystem of frameworks, and we’re contributing to that by building mavro — a fast, minimalistic, and scalable Node.js framework built on Koa and TypeScript. 
We’re actively contributing  shaping mavro to be the go-to framework for developers who value simplicity and performance. If you’re a Node.js enthusiast looking to contribute to exciting open-source projects, we’d love to collaborate! Let’s push the boundaries of backend development together! 👉 [Join our community on Discord](https://discord.gg/jJK5kZx84B)
